akka-docs-dev/rst/scala/code/docs/stream/ReactiveStreamsDocSpec.scala

```diff
@@ -135,4 +133,16 @@ class ReactiveStreamsDocSpec extends AkkaSpec {
     assertResult(storage)
   }
 
+  "use a processor" in {
+
+    //#use-processor
+    // An example Processor factory
+    def createProcessor: Processor[Int, Int] = Flow[Int].toProcessor.run()
+
+    val src: Source[Int, Unit] = Source.single(42).viaProcessor(() => createProcessor)
+    val flow: Flow[Int, Int, Unit] = Flow[Int].viaProcessor(() => createProcessor)
```

Comment:
Well, I still think "transform" and "viaProcessor" are the best options so far. I don't like calling "transform" viaStage, because it hides the intent of it being a generic stream transformation operation. You can argue that "viaProcessor" is also a generic stream transformation (in fact it is more generic), but that is not the preferred one. Calling it "process" invites trouble, because of the inevitable question of "what is the difference between process and transform? Which one should I use?"